clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -383,23 +383,17 @@ protected boolean isStarted() {
     try {
       if (configuration.isAutoCreate()) {
         try {
-          this.entity.validateCache(cacheId, clientStoreConfiguration);
-        } catch (ClusteredTierValidationException ex) {
-          if (ex.getCause() instanceof InvalidStoreException) {
-            try {
-              this.entity.createCache(cacheId, clientStoreConfiguration);
-            } catch (TimeoutException e) {
-              throw new CachePersistenceException("Unable to create clustered tier proxy '"
-                  + cacheIdentifier.getId() + "' for entity '" + entityIdentifier
-                  + "'; create operation timed out", e);
-            }
-          } else {
-            throw ex;
+          entity.createCache(cacheId, clientStoreConfiguration);
+        } catch (ClusteredTierCreationException e) {
+          // An InvalidStoreException means the cache already exists. That's fine, the validateCache will then work
+          if (!(e.getCause() instanceof InvalidStoreException)) {
+            throw e;
           }
         }
-      } else {
-        this.entity.validateCache(cacheId, clientStoreConfiguration);
       }
+
+      entity.validateCache(cacheId, clientStoreConfiguration);
```

Comment:
If a client has successfully created the cache with a store config, there is no need to validate it with the same store config. The whole point of validation of cache is to validate the existing store at server with client provided config, since creation just succeeded with that, validation of cache is not required.